src/test/java/fr/inria/coming/spoon/repairability/TestUtills.java

```diff
@@ -44,4 +44,27 @@ public static FinalResult runRepairability(String toolName, String inputFiles) t
         assertNotNull(result);
         return result;
     }
+
+    public static FinalResult runRepairabilityPrint(String toolName, String inputFiles) throws Exception {
+        ComingMain cm = new ComingMain();
+
+        FinalResult result = cm.run(
+                new String[]{"-mode",
+                        "repairability",
+                        "-repairtool",
+                        toolName,
+                        "-input",
+                        "git",
+                        "-output",
+                        "./out",
+                        "-filter",
+                        "maxfiles",
+                        "-parameters",
+                        "max_files_per_commit:1",
+                        "-location",
+                        TestUtills.class.getResource(inputFiles).getFile()});
```

Comment:
Sorry! Earlier I got confused. 

We don't need to `getResource` here. Just `inputFiles` in place of ` TestUtills.class.getResource(inputFiles).getFile()` will work fine.

Reply:
No, I have actually tested this but it doesn't change anything.
you are right about the commons-lang and CI having an error, but it doesn't even work on my local.

Reply:
Just to be sure: 
- the inputFile was `repogit4testv0`
- `repogit4testv0` was present in the root folder of the coming repo ?

It worked on my system.

Reply:
sorry!! I had /repogit4testv0 Everything is fine now, thanks!

Reply:
yay!